src/network/networkd-link.c

```diff
@@ -1808,41 +1810,39 @@ int link_up(Link *link) {
         if (r < 0)
                 return log_link_error_errno(link, r, "Could not open IFLA_AF_SPEC container: %m");
 
-        if (link_ipv6_enabled(link)) {
-                /* if the kernel lacks ipv6 support setting IFF_UP fails if any ipv6 options are passed */
```

Comment:
I think dropping the check is dangerous.

Reply:
Actually I dropped I missed that comment.  thanks